components/ImageWithBlur.tsx

```diff
@@ -0,0 +1,19 @@
+import Image, { ImageProps } from "next/image";
+import { getBase64Url } from "@/lib/plaiceholder";
+
+export default async function ImageWithBlur({
+  imageUrl,
```

Comment:
We could just reuse the `src` prop itself right?

Reply:
No, because src could also be the type of StaticImport

Reply:
I think i can have different return for different type of src

Reply:
I merge this branch with main and I am getting this error
⨯ The requested resource isn't a valid image for /logo.webp received text/html; charset=utf-8
ImageError: The requested resource isn't a valid image.
    at imageOptimizer (/workspaces/leaderboard/node_modules/.pnpm/next@14.1.3_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/image-optimizer.js:635:19)
    at DevServer.imageOptimizer (/workspaces/leaderboard/node_modules/.pnpm/next@14.1.3_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:605:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at runNextTicks (node:internal/process/task_queues:64:3)
    at listOnTimeout (node:internal/timers:540:9)
    at process.processTimers (node:internal/timers:514:7)
    at async cacheEntry.imageResponseCache.get.incrementalCache (/workspaces/leaderboard/node_modules/.pnpm/next@14.1.3_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/next-server.js:179:65)
    at async /workspaces/leaderboard/node_modules/.pnpm/next@14.1.3_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/response-cache/index.js:90:36
    at async /workspaces/leaderboard/node_modules/.pnpm/next@14.1.3_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/lib/batcher.js:41:32 {
  statusCode: 400
}

Reply:
I tried to merge but this error is stopping me from doing a pnpm build